curriculum/challenges/english/07-scientific-computing-with-python/learn-string-manipulation-by-building-a-cipher/65688a50e6c998a21d8e41d3.md

```diff
@@ -14,7 +14,7 @@ my_string = 'brain'
 my_string[0] = 't'
 ```
 
-Confirm that by using the bracket notation to access the first letter in `text` and try to change it into a character of your choice. You will see the ouput disappear.
+Confirm that by using the bracket notation to access the first letter in `text` and try to change it into a character of your choice. You will see the ouput disappear and a traceback appear.
```

Comment:
We have never mentioned what a traceback is. We should do it before this step or simply say "error" here.